overlay.py

```diff
@@ -54,6 +56,11 @@ def __init__(self, bike, width=1280, height=740, bg: str=None):
 
 		self.start_time = time.time()
 
+	def publish_errors(self, message: dict) -> None:
+		message['camera'] = self.device
```

Comment:
While we're here, can we also add additional details here too? Although the trace is informative, it would be good to have an idea of the current state of the overlay when it made those errors. This will lead to a faster debugging time for the engineers.
- `self.backend_name`
- `bike_version`
- `self.bg_path` if it is set
- `configs`

We should also have a `message` key with an easy to read message so that any subscriber to the error topic can just do `error.message`

There may be more that I might be missing and we might not even need the above. Let me know what you think.

Reply:
What should the message say?

Reply:
Should configs be accessed using `read_configs()`? Because right now, it is not an instance variable nor class variable.

Reply:
> What should the message say?

Maybe try to get it via `e.message` See: https://www.semicolonworld.com/question/59435/how-to-get-exception-message-in-python-properly
if the exception doesn't have a `message` just print the whole trace?

>Should configs be accessed using read_configs()? Because right now, it is not an instance variable nor class variable.

I think `read_configs()` is fine.

Reply:
Was able to append the configs to the JSON message just fine. 

On the other hand, not sure if the link is outdated but doing `e.message` doesn't seem to work. In case you were curious, simply printing `e` is just fine. For example:
```
>>>try:
>>>    raise Exception("Test error")
>>>except Exception as error_message:
>>>    print(error_message)
Test error
```